<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>queryView</class>
 <widget class="QDialog" name="queryView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>254</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>3</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QToolButton" name="btnExport">
     <property name="text">
      <string>Export</string>
     </property>
     <property name="icon">
      <iconset resource="../res/res.qrc">
       <normaloff>:/res/image/icon/219.png</normaloff>:/res/image/icon/219.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QTabWidget" name="tabWidget">
     <widget class="QTableView" name="tab">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="cornerButtonEnabled">
       <bool>true</bool>
      </property>
      <property name="modelColumn" stdset="0">
       <number>2</number>
      </property>
      <attribute name="title">
       <string>Data</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="cmbViewFmt">
     <item>
      <property name="text">
       <string>ascii</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>hex</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>char</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>short</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>int</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>float</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>double</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>unsigned char</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>unsigned short</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>unsigned int</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Format</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QLabel" name="labComment">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/res.qrc"/>
 </resources>
 <connections/>
</ui>
